assets/source/setup-guide/app/helpers/documentation-link-props.js

```diff
@@ -21,18 +33,19 @@ import { recordEvent } from '@woocommerce/tracks';
  *
  * Please be careful not to overwrite the `onClick` handler coincidently.
  *
- * @fires wcadmin_pfw_documentation_link_click on click, with given `linkId` and `context`.
+ * Notice documentation_link_click is the default eventName and pfw_ is not required
```

Comment:
This line seems no longer be applicable. We do need `pfw_` prefix, and the default value is stated below.
```suggestion
```